src/components/BookAdd.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+function BookAdd() {
+  return (
+    <>
+      <h2>ADD NEW BOOK</h2>
+      <input type="text" placeholder="Book Title .." />
+      <form action="#">
```

Comment:
- [ ] It will be great to have both fields for the book. That is the title and the author of the book. Great work adding the title field, now add the author field also. This is a requirement before the project can be approved.

Reply:
Hello @peterrobert I looked at the requirements for this project and what you asking me to implement is meant for the next project. Im only supposed to setup the project with redux and initialize it with components only. I'd really appreciate it if you had a rethink of the changes you asked me to make because they can be done in next project. Thanks

Reply:
@KayLemba thanks for reaching out and we have solved it.

Reply:
Done